app/models/user.rb

```diff
@@ -1,18 +1,19 @@
 class User < ApplicationRecord
   SPECIAL = "?<>',?[]}{=-)(*&^%$#`~{}!"
   REGEX = /[#{SPECIAL.gsub(/./){|char| "\\#{char}"}}]/
-  ROLLER = { user: 0, admin: 1 }
+  ROLLER = { user: 0, admin: 1, invited_user: 2 }
```

Comment:
This looks like the roles hash and invited_user doesn't seem like a role. Can we keep it separate?